.editorconfig

```diff
@@ -0,0 +1,20 @@
+[{*.kt,*.kts}]
+ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
+
+max_line_length = 180
+
+ij_kotlin_line_comment_at_first_column = false
+ij_kotlin_line_comment_add_space = true
+
+ij_kotlin_name_count_to_use_star_import = 2147483647
+ij_kotlin_name_count_to_use_star_import_for_members = 2147483647
+
+ij_kotlin_keep_blank_lines_in_declarations = 1
+ij_kotlin_keep_blank_lines_in_code = 1
+ij_kotlin_keep_blank_lines_before_right_brace = 0
+
+ij_kotlin_align_multiline_parameters = false
+ij_continuation_indent_size = 4
+
+ij_kotlin_import_nested_classes = false
+ij_kotlin_imports_layout = *, ^
```

Comment:
Please add configuration for `end_of_line` and `insert_final_newline` for all files

Reply:
I added: 
```
insert_final_newline = true
end_of_line = crlf
```
It's ok?

Reply:
Note "for all files", not only the kotlin one

Reply:
1. Please check once again which settings I was referring to in first comment and which you have placed in "[*]" section
2. For `end_of_line` use `lf`

Reply:
1. Yes, my bad I swapped  `indent_style, indent_size` with `end_of_line, insert_final_newline` 
2. I used `lf``

Reply:
Last one small improvement: put `indent_*` settings on top of the kotlin related settings. Just to group them :)